docker/requirements.txt

```diff
@@ -1,12 +1,14 @@
 --extra-index-url="https://packages.dea.ga.gov.au"
+git+https://github.com/ubarsc/rios.git
 git+https://github.com/digitalearthafrica/deafrica-sandbox-notebooks.git#subdirectory=Tools
 datacube[performance,s3]==1.8.4.dev81+g80d466a2
-click<8.0.0
-morecantile==1.2.0
+dask_ml==1.9.0
+click==8.0.1
+morecantile==2.1.2
 odc_stats
-aiobotocore[boto3,awscli]==1.0.7
-pyyaml==5.3.1
-botocore==1.15.32
+aiobotocore==1.3.1
+pyyaml==5.4.1
+botocore==1.20.49
 s3transfer==0.3.7
-boto3==1.12.30
-GDAL
+boto3==1.17.49
+GDAL==3.3.0
```

Comment:
I think you should remove all specific requirements. Requirements are set with the constraints file now.